{"name": "test_login_failed_user_password_wrong[case0]", "status": "passed", "parameters": [{"name": "case", "value": "{'user': '', 'password': 'qqqqqqqq', 'check': '缺少用户名或密码'}"}], "start": 1665209568837, "stop": 1665209569365, "uuid": "12df0a6b-ccc8-4402-976a-a8c2506b70a3", "historyId": "4aa489bf5e13b73561afb68efc18c377", "testCaseId": "253d57a3ffef1d7866e3686ffe2eb276", "fullName": "TestCases.test_login.TestLogin#test_login_failed_user_password_wrong", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init', 'back_to_login_page')"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-E380NSU"}, {"name": "thread", "value": "18584-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}